common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -0,0 +1,752 @@
+                    a2=int(ans[1])
+                except ValueError:
+                    a1=0
+                    a2=0
+                return (a1+a2)==int(expect)
+        """),
+    }
+    status_indicators = {
+        'correct': ['div.correct'],
+        'incorrect': ['div.incorrect'],
+        'unanswered': ['div.unanswered', 'div.unsubmitted'],
+    }
+
+    def setUp(self, *args, **kwargs):
+        """
+        Additional setup for ScriptProblemTypeTest
+        """
+        super(ScriptProblemTypeTest, self).setUp(*args, **kwargs)
+        self.problem_page.a11y_audit.config.set_rules({
+            'ignore': [
+                'label',  # TODO: AC-287
+            ]
+        })
+
+    def answer_problem(self, correct):
+        """
+        Answer script problem.
+        """
+        # Correct answer is any two integers that sum to 10
+        first_addend = random.randint(-100, 100)
+        second_addend = 10 - first_addend
+
+        # If we want an incorrect answer, then change
+        # the second addend so they no longer sum to 10
+        if not correct:
+            second_addend += random.randint(1, 10)
+
+        self.problem_page.fill_answer(first_addend, input_num=0)
+        self.problem_page.fill_answer(second_addend, input_num=1)
+
+
+class CodeProblemTypeTest(ProblemTypeTestBase, ProblemTypeTestMixin):
+    """
+    TestCase Class for Code Problem Type
+    """
+    problem_name = 'CODE TEST PROBLEM'
+    problem_type = 'code'
+
+    factory = CodeResponseXMLFactory()
+
+    factory_kwargs = {
+        'question_text': 'Submit code to an external grader',
+        'initial_display': 'print "Hello world!"',
+        'grader_payload': '{"grader": "ps1/Spring2013/test_grader.py"}',
+    }
+
+    status_indicators = {
+        'correct': ['.grader-status .correct ~ .debug'],
+        'incorrect': ['.grader-status .incorrect ~ .debug'],
+        'unanswered': ['.grader-status .unanswered ~ .debug'],
+    }
+
+    def setUp(self, *args, **kwargs):
+        """
+        Additional setup for CodeProblemTypeTest
+        """
+        super(CodeProblemTypeTest, self).setUp(*args, **kwargs)
+        self.problem_page.a11y_audit.config.set_rules({
+            'ignore': [
+                'color-contrast',  # TODO: AC-286
+                'label',  # TODO: AC-286
+            ]
+        })
+
+    def answer_problem(self, correct):
+        """
+        Answer code problem.
+        """
+        # The fake xqueue server is configured to respond
+        # correct / incorrect no matter what we submit.
+        # Furthermore, since the inline code response uses
+        # JavaScript to make the code display nicely, it's difficult
+        # to programatically input text
+        # (there's not <textarea> we can just fill text into)
+        # For this reason, we submit the initial code in the response
+        # (configured in the problem XML above)
+        pass
+
+    def test_answer_incorrectly(self):
```

Comment:
No, this overrides the inherited version of `test_answer_incorrectly`.   For `CodeProblemTypeTest`,  this test case will just pass.